### DISCUSSION TITLE
Unable to ffmpeg to live stream endpoint

### DISCUSSION THREAD

**Discussion by philipmather** [tsightler/ring-mqtt#414] [Support Q&A]
Manual install on a pi3...
```homeassistant@home:~$ node --version
v16.16.0
homeassistant@home:~$ rtsp-simple-server --version
v0.19.2
homeassistant@home:~$ apt list mosquitto-clients
Listing... Done
mosquitto-clients/stable,now 2.0.11-1 armhf [installed]
homeassistant@home:~$ jq --version
jq-1.6
```

Notifications of motion on the gen 2 ring doorbell work fine. Trying `ffmpeg -i rtsp://127.0.0.1:8554/343ea41caf7c_live -c copy output.mp4` to get the video I'm given a 404?
```
homeassistant@home:~$ ffmpeg -i rtsp://127.0.0.1:8554/343ea41caf7c_live -c copy output.mp4
...
[rtsp @ 0x1cbacd0] method DESCRIBE failed: 404 Not Found
rtsp://127.0.0.1:8554/343ea41caf7c_live: Server returned 404 Not Found
```

Hence I get the same when I go to configure the generic camera component?
If I go to HA > Configuration > Devices and switch the live stream option on manually it moves back a split second later and I get the following in ring-mqtt.log... 

```2022-08-01T22:28:01.362Z ring-mqtt [Front Door] Starting a keepalive stream for camera
2022-08-01T22:28:01.458Z ring-mqtt [Front Door] The keepalive stream has started
2022-08-01T22:28:01.494Z ring-mqtt [Front Door] The keepalive stream has stopped
2022-08-01T22:28:09.519Z ring-mqtt WARNING - Unhandled Promise Rejection
TypeError: Cannot read properties of undefined (reading 'sourceReady')
    at Camera.startKeepaliveStream (/opt/ring-mqtt/devices/camera.js:747:30)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```
Any pointers appreciated!

**Comment by tsightler** [comment]
Make sure you are not starting rtsp-simple-server manually (it must be started by ring-mqtt) and also check the logs to make sure you see that rss is being started and that the paths are being registered.  But the best suggestion is to use the Docker version.
